tools/engine_tool/lib/src/commands/command.dart

```diff
@@ -53,3 +53,13 @@ void addConfigOption(
     },
   );
 }
+
+/// Adds the -j option to the parser.
+void addDashJOption(ArgParser parser) {
+  parser.addOption(
+    dashJFlag,
```

Comment:
Nit: since `-j` is also the abbreviation, consider making this flag's full name `concurrency` to improve discoverability

Reply:
Done.